karbon/mocks/TestKarbonCreateClusterRegistrationAndAddonSetInfo.yaml

```diff
@@ -0,0 +1,335 @@
+version: api.keploy.io/v1beta2
+kind: Http
+name: TestKarbonCreateClusterRegistrationAndAddonSetInfo
+spec:
+    metadata:
+        Body: ""
+        Header: map[Accept:[application/json] Content-Type:[application/json] User-Agent:[nutanix]]
+        Proto: HTTP/1.1
+        ProtoMajor: "1"
+        ProtoMinor: "1"
+        URL: https://10.45.30.138:9440/karbon/v1-alpha.1/k8s/cluster-registrations/634f5852-ce1b-465c-a95a-9b8dffbdde42
```

Comment:
please swap out the IPs here (and elsewhere) with a dummy value e.g. prism-test.nutanix.com.